### ISSUE TITLE
Telescope workflow implementation: DOAB

### ISSUE THREAD

**Issue by aroelo** [The-Academic-Observatory/observatory-platform#304]
See https://docs.google.com/document/d/14y7lRHYr3MxGzTsiP3j60t7DOJ1tsJLLAMMhQeDHNMA/edit#heading=h.mbrtwfqv96yi for background information

There is OAI harvesting and a csv file available, there seem to be small differences between these two outputs.
We could combine the two or choose to only use one of them.

**Comment by aroelo** [comment]
@alkimozaygen @rhosking
I am updating the telescope to use our template and found out that DOAB made quite some changes in how their data is available.
The OAI PMH harvesting url is changed. Further usage seems to be the same.

The CSV file is completely changed.
Before it looked like this:
![image](https://user-images.githubusercontent.com/22207600/109105584-35093780-7769-11eb-925c-c498093e5a58.png)

And now it looks like this:
![image](https://user-images.githubusercontent.com/22207600/109105667-5c600480-7769-11eb-873d-a7d0a2a89c3d.png)

Which is very similar to the OAPEN metadata csv format. (I actually got confused and had to look twice that I had the right file, the file name is also the same)

With this updated CSV most information that we first obtained by OAI PMH harvesting is now in the CSV as well (as far as I can tell all fields with the exception of setSpec).

I propose we change the schema of DOAB and only use the CSV file. Because this is similar to OAPEN metadata it will be easier to develop/maintain and might also be easier to handle downstream.
